.circleci/config.yml

```diff
@@ -34,8 +34,8 @@ jobs:
         name: Make
         command: make check
     - run:
-        name: mock make package
-        command: make BUNDLE_DIR=/tmp MOCK_BUNDLE=true package
+        name: mock make out/macos-amd64/crc-installer-macos-amd64.pkg
+        command: make NO_CODESIGN=1 BUNDLE_DIR=/tmp MOCK_BUNDLE=true out/macos-amd64/crc-installer-macos-amd64.pkg
```

Comment:
Do we really need `installer` as part of naming doesn't `crc-macos-amd64.pkg` good enough since `.pkg` format is the installer one?

Reply:
good point I remove it

Reply:
@guillaumerose looks like you forgot to update it in the circle CI configuration since it still present.

Reply:
thanks!